server/src/test-fast/java/com/thoughtworks/go/server/service/AgentConfigServiceTest.java

```diff
@@ -1,106 +0,0 @@
-/*
- * Copyright 2019 ThoughtWorks, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.thoughtworks.go.server.service;
-
-import com.thoughtworks.go.config.AgentConfig;
-import com.thoughtworks.go.config.CruiseConfig;
-import com.thoughtworks.go.config.GoConfigDao;
-import com.thoughtworks.go.config.update.AgentsEntityConfigUpdateCommand;
-import com.thoughtworks.go.domain.AgentInstance;
-import com.thoughtworks.go.domain.AgentRuntimeStatus;
-import com.thoughtworks.go.remote.AgentIdentifier;
-import com.thoughtworks.go.server.dao.DatabaseAccessHelper;
-import com.thoughtworks.go.server.domain.AgentInstances;
-import com.thoughtworks.go.server.domain.Username;
-import com.thoughtworks.go.server.service.result.HttpLocalizedOperationResult;
-import com.thoughtworks.go.util.SystemEnvironment;
-import com.thoughtworks.go.util.TriState;
-import org.junit.Before;
-import org.junit.Test;
-import org.mockito.ArgumentCaptor;
-
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.*;
-
-public class AgentConfigServiceTest {
-
-    private GoConfigService goConfigService;
-    private AgentConfigService agentConfigService;
-    private EnvironmentConfigService environmentConfigService;
-
-    @Before
-    public void setUp() throws Exception {
-        goConfigService = mock(GoConfigService.class);
-        environmentConfigService = mock(EnvironmentConfigService.class);
-        agentConfigService = new AgentConfigService(goConfigService);
-    }
-
-    @Test
-    public void shouldEnableAgentWhenPending() {
```

Comment:
Can you please check if these removed tests are present in `AgentServiceTest`? because we moved a few methods from `AgentConfigService` to `AgentService`

Reply:
Verified and added in [commit](https://github.com/gocd/gocd/pull/6271/commits/c8615c125caa2dd502de86ec9301187f5b2dd193)